R/system.R

```diff
@@ -601,12 +601,47 @@ getDBConnection <- function(type, host = NULL, port = "", databaseName = "", use
                                      password = password, host = host, port = port)
       connection_pool[[key]] <- conn
     }
-  } else if (type == "presto") {
+  } else if (type == "presto" || type == "treasuredata") {
     if(!requireNamespace("DBI")){stop("package DBI must be installed.")}
     if(!requireNamespace("RPresto")){stop("package Presto must be installed.")}
-    loadNamespace("RPresto")
-    drv <- RPresto::Presto()
-    conn <- RPresto::dbConnect(drv, user = username, password = password, host = host, port = port, schema = schema, catalog = catalog, session.timezone = Sys.timezone(location = TRUE))
+    # use the same key "presto" for presto and treasuredata since they both use
```

Comment:
The comment sentence is ending in the middle.

Reply:
Updated comment.